Flaky test: `pymc/tests/step_methods/hmc/test_nuts.py::TestNUTSNormal::test_kstest`

It failed in [this pipeline](https://github.com/pymc-devs/pymc/actions/runs/3332747252/jobs/5513973833) with error:

```
================================== FAILURES ===================================
_________________________ TestNUTSNormal.test_kstest __________________________

self = <pymc.tests.step_methods.hmc.test_nuts.TestNUTSNormal object at 0x0000022B4908E310>

    def test_kstest(self):
        for varname, cdf in self.cdfs.items():
            samples = self.samples[varname]
            if samples.ndim == 1:
                t, p = stats.kstest(samples[:: self.ks_thin], cdf=cdf)
                assert self.alpha < p
            elif samples.ndim == 2:
                pvals = []
                for samples_, cdf_ in zip(samples.T, cdf):
                    t, p = stats.kstest(samples_[:: self.ks_thin], cdf=cdf_)
                    pvals.append(p)
                t, p = stats.combine_pvalues(pvals)
>               assert self.alpha < p
E               AssertionError

pymc\tests\sampler_fixtures.py:58: AssertionError
---------------------------- Captured stdout setup ----------------------------
\u2588\r\r |----------------------------------------| 0.00% [0/22000 00:00<? Sampling 2 chains, 0 divergences]\r\r |------------------------------------| 0.00% [1/22000 00:00<00:00 Sampling 2 chains, 0 divergences]\r\r |------------------------------------| 0.01% [2/22000 00:00<00:00 Sampling 2 chains, 0 divergences]\r\r |------------------------------------| 0.01% [3/22000 00:00<00:00 Sampling 2 chains, 0 divergences]\r\r |------------------------------------| 0.02% [4/22000 00:00<00:00 Sampling 2 chains, 0 divergences]\r\r |------------------------------------| 0.02% [5/22000 00:00<00:00 Sampling 2 chains, 0 divergences]\r\r |\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 100.00% [22000/22000 00:17<00:00 Sampling 2 chains, 0 divergences]\r
---------------------------- Captured stderr setup ----------------------------
Initializing NUTS using jitter+adapt_diag...
Multiprocess sampling (2 chains in 2 jobs)
NUTS: [a]
Sampling 2 chains for 1_000 tune and 10_000 draw iterations (2_000 + 20_000 draws total) took 27 seconds.
----------------------------- Captured log setup ------------------------------
INFO     pymc:sampling.py:2479 Initializing NUTS using jitter+adapt_diag...
INFO     pymc:sampling.py:606 Multiprocess sampling (2 chains in 2 jobs)
INFO     pymc:sampling.py:244 NUTS: [a]
INFO     pymc:sampling.py:678 Sampling 2 chains for 1_000 tune and 10_000 draw iterations (2_000 + 20_000 draws total) took 27 seconds.
```

--- a/pymc/tests/sampler_fixtures.py
+++ b/pymc/tests/sampler_fixtures.py
@@ -149,6 +149,7 @@ class BaseSampler(SeededTest):
                 cores=cls.chains,
                 return_inferencedata=False,
                 compute_convergence_checks=False,
+                random_seed=cls.random_seed,
             )
         cls.samples = {}
         for var in cls.model.unobserved_RVs:

